tapdb/universe_federation.go

```diff
@@ -328,11 +333,15 @@ func (u *UniverseFederationDB) QueryFederationSyncConfigs(
 			return err
 
 		default:
-			// Parse global db sync configs.
-			globalConfigs = make(
-				[]*universe.FedGlobalSyncConfig,
-				len(globalDbConfigs),
-			)
+			// Start with the default global sync configs.
+			for i := range defaultGlobalSyncConfigs {
+				config := defaultGlobalSyncConfigs[i]
+
+				globalConfigSet[config.ProofType] = config
```

Comment:
Ah ok, so it's for if a user sets one, and not the other. 

We should have a unit test here, should be at most 20 lines.